content/blog/2020/05/2020-05-07-docker-windows-agents.adoc

```diff
@@ -0,0 +1,121 @@
+---
+layout: post
+title: "Windows Docker Agent Images"
+tags:
+- announcement
+- docker
+- platform-sig
+author: slide_o_mix
+opengraph:
+  image: /images/docker/dockerJenkins_social.png
+---
+
+We would like to announce the availability of Windows agent images for Docker.
+
```

Comment:
```suggestion
These images allow provisioning Jenkins agents with Windows OS on Docker and Kubernetes.

```

An example of using an image in Jenkins Pipeline would be great., e.g. for inbound-agent below